lib/jazzy/jazzy_markdown.rb

```diff
@@ -13,10 +13,9 @@ def header(text, header_level)
                       .sub(/^-/, '')
                       .sub(/-$/, '')
 
-      "<a href='##{text_slug}' class='anchor' aria-hidden=true>" \
-        '<span class="header-anchor"></span>' \
```

Comment:
Is it possible that some themes display this anchor as a link icon when hovering over the header, to make it easier to share a link to a particular section?